.github/workflows/ci-cd-backend.yml

```
@@ -59,40 +59,50 @@ jobs:
-          
-          # 8080 포트를 사용하는 프로세스 종료
-          # lsof 명령어를 사용하여 8080 포트를 점유하고 있는 프로세스의 PID를 찾습니다.
+          # Spring WAS (:8080) 종료
           PID=$(lsof -t -i:8080 || true)
           if [ -n "$PID" ]; then
-            echo "Killing process with PID: $PID that is using port 8080"
-            kill -9 $PID
-            sleep 5 # 프로세스 종료 대기
+            echo "Found running process with PID: $PID. Sending SIGTERM for graceful shutdown."
+            kill -SIGTERM $PID
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)